edx_proctoring/api.py

```diff
@@ -1362,7 +1362,7 @@ def get_active_exams_for_user(user_id, course_id=None):
     """
     result = []
 
-    student_active_exams = ProctoredExamStudentAttempt.objects.get_active_student_attempts(user_id, course_id)
+    student_active_exams = ProctoredExamStudentAttempt.objects.get_active_student_attempts(user_id, course_id, exam_id)
```

Comment:
Above you have added `exam_id=None`. So, can we add a handler to return if no `exam_id` is passed ?